src/main/kotlin/inc/evil/d17map/controllers/ReservationController.kt

```diff
@@ -0,0 +1,46 @@
+package inc.evil.d17map.controllers
+
+import inc.evil.d17map.dtos.ReservationDto
+import inc.evil.d17map.services.ReservationService
+import jakarta.persistence.EntityNotFoundException
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.time.LocalDate
+import java.util.*
+
+@RestController
+@RequestMapping("/reservations")
+class ReservationController(private val reservationService: ReservationService) {
+    @GetMapping
+    fun getReservationsByDay(@RequestParam("day") day: String?): ResponseEntity<Any> {
+        if (day == null) {
+            return ResponseEntity("Query parameter 'day' must be specified", HttpStatus.BAD_REQUEST)
+        }
+
+        val parsedDate = try {
```

Comment:
Nie dałoby się wskazać Springowi, żeby automatycznie parsował datę? Chodzi mi o to, aby zrobić coś w stylu:
`(@RequestParam("day") day: Date?)